docs/orchestrating-elastic-stack-applications/accessing-elastic-services.asciidoc

```diff
@@ -148,10 +149,22 @@ Create a Kubernetes secret with:
 
 [source,sh]
 ----
-kubectl create secret generic my-cert --from-file=ca.crt=tls.crt --from-file=tls.crt=tls.crt --from-file=tls.key=tls.key
+kubectl create secret generic my-cert --from-file=ca.crt --from-file=tls.crt --from-file=tls.key
```

Comment:
I was honestly confused by the original example. Why would one want to specify the leaf certificate as the CA certificate? If the leaf certificate were also the CA at the same time we would not need to specify it. So I am thinking the original example was done by mistake. At the very least it would have required some additional explanation. So I made this change on purpose to reflect what I would think is a typical use case.